lang/cs/Org.Apache.REEF.Network.Tests/GroupCommunication/GroupCommunicationTests.cs

```diff
@@ -107,22 +110,21 @@ public void TestBroadcastReduceOperators()
             int fanOut = 2;
 
             var groupCommunicationDriver = GetInstanceOfGroupCommDriver(driverId, masterTaskId, groupName, fanOut, numTasks);
-
             ICommunicationGroupDriver commGroup = groupCommunicationDriver.DefaultGroup
-                .AddBroadcast<int>(
-                    broadcastOperatorName,
-                    masterTaskId,
-                    TopologyTypes.Flat,
-                    GetDefaulCodecConfig(),
-                    GetDefaulDataConverterConfig())
-                .AddReduce<int>(
-                    reduceOperatorName,
-                    masterTaskId,
-                    TopologyTypes.Flat,
-                    GetDefaulCodecConfig(),
-                    GetDefaulDataConverterConfig(),
-                    GetDefaulReduceFuncConfig())
-                .Build();
+                    .AddBroadcast<int>(
```

Comment:
This changes only the formatting, right? If so, please remove the change to keep this simple.